src/test/kotlin/org/rust/lang/core/completion/RustStdlibCompletionTest.kt

```diff
@@ -0,0 +1,32 @@
+package org.rust.lang.core.completion
+
+import com.intellij.testFramework.LightProjectDescriptor
+import org.assertj.core.api.Assertions
+import org.rust.cargo.util.cargoProject
+import org.rust.lang.core.resolve.RustMultiFileResolveTestCaseBase
+
+class RustStdlibCompletionTest : RustMultiFileResolveTestCaseBase() {
+    override val dataPath = "org/rust/lang/core/completion/fixtures/stdlib"
+
+    override fun getProjectDescriptor(): LightProjectDescriptor = WithStdlibRustProjectDescriptor()
+
+    fun testHasStdlibSources() {
```

Comment:
I think there is no need to duplicate this test both in completion and in resovle.